### ISSUE TITLE
Project Rewrite

### ISSUE THREAD

**Issue by ZiQiLN** [humwrk/humwrk#10]
Damn... Welcome to Humwrk - that's like third rewrite of project caused by messy vision of @ZiQiLN. Apparently, now we'll focus on building product with single responsibility and frequent updates instead building big one, so in other worlds we'll be putting together small pieces of shit instead releasing one big piece of shit lol

**Comment by emadbaqeri** [comment]
hey mate @keinsell, it's been long that you are not active and not working on the project. can we start the project and build this project up? waiting for your response.

**Comment by emadbaqeri** [comment]
@keinsell I'm also okay working on front-end with react and typescript or mobile app with react-native

**Comment by keinsell** [comment]
Yeah mate, that's good idea but currently I'm in NL and I forgot my MacBook, coming back to my home office at end of this month and with a bit different convention I was planing to make use of humwrk. As Araclx Mello where Humwrk will be part of it as a professional service part.


On June 6, 2021 at 11:00 PM, Emad Baqeri ***@***.***) wrote:
@keinsell ( https://github.com/keinsell ) I'm also okay working on front-end with react and typescript or mobile app with react-native

**Comment by emadbaqeri** [comment]
Good to hear from you, I'm waiting for you response at the end of the month, and I should mention that I can work on any opensource ideas that you have in mind almost full-time currently

**Comment by keinsell** [comment]
Awesome, are you into Ionic framework? it's one of our main technology in projects where we're also doing mobile apps.


On June 7, 2021 at 9:34 AM, Emad Baqeri ***@***.***) wrote:
Good to hear from you, I'm waiting for you response at the end of the month, and I should mention that I can work on any opensource ideas that you have in mind almost full-time currently

**Comment by emadbaqeri** [comment]
Nope, but I can put some time on it for understanding the concepts and learning.

**Comment by keinsell** [comment]
> Nope, but I can put some time on it for understanding the concepts and learning. 

Basically it's like React.js but you can implement mobile native-like components that interact with OS API.

We'll be using it to create an mobile app.

**Comment by emadbaqeri** [comment]
Aha alright, I'm going to give it a try as I'm already familiar with react and react-native I think it would be fun woking with that
